wifiphisher/pywifiphisher.py

```diff
@@ -81,6 +81,11 @@ def parse_args():
         nargs='+',
         help=("Specify the interface(s) that will have their connection protected (i.e. NetworkManager will be prevented from controlling them). " +
               "Example: -pI wlan1 wlan2"))
+    parser.add_argument(
+        "-m",
+        "--mitminterface",
```

Comment:
I suppose there are some options that cannot be used together with `-m` (e.g. `-iI`). You should cover this by adding some checks on `opmode.py` (see `_check_args()`).

Reply:
Since I am intentionally handling `-mI` the same as `-iI`(the assignment is performed before opmode's initialisation), the only conflict not already mentioned should be between the two internet interfaces (`-mI` and `-iI`). I added a message in `check_args()` to warn the user that `-mI` will always take precedence in case both are used, as in the end it is `-iI` that gets overwritten if `-mI` is present.  

If this does not sound correct I could maybe modify the parsing in order to ensure that we will always keep the last argument provided between the two while performing the MITM regardless.
i.e.  `wifiphisher -mI wlan1 ... -iI wlan3 ...` would be the same as `wifiphisher -mI wlan3 ...`